dashboard/config/foorm/forms/surveys/pd/summer_workshop_post_survey.json

```diff
@@ -0,0 +1,476 @@
+     "columns": [
+      {
+       "value": "1",
+       "text": "Strongly Disagree"
+      },
+      {
+       "value": "2",
+       "text": "Disagree"
+      },
+      {
+       "value": "3",
+       "text": "Slightly Disagree"
+      },
+      {
+       "value": "4",
+       "text": "Neutral"
+      },
+      {
+       "value": "5",
+       "text": "Slightly Agree"
+      },
+      {
+       "value": "6",
+       "text": "Agree"
+      },
+      {
+       "value": "7",
+       "text": "Strongly Agree"
+      }
+     ],
+     "rows": [
+      {
+       "value": "I found the activities we",
+       "text": "I found the activities we did in this workshop interesting and engaging."
+      },
+      {
+       "value": "I was highly active and p",
+       "text": "I was highly active and participated a lot in the workshop activities."
+      },
+      {
+       "value": "When I'm not in Code.org ",
+       "text": "When I'm not in Code.org workshops, I frequently talk about ideas or content from the workshop with others."
+      },
+      {
+       "value": "I am definitely planning ",
+       "text": "I am definitely planning to make use of ideas and content covered in this workshop in my classroom."
+      }
+     ]
+    },
+    {
+     "type": "matrix",
+     "name": "curriculum_impressions",
+     "title": "Having been through this workshop, what are your overall impressions of the {workshopCourse} curriculum?",
+     "columns": [
+      {
+       "value": "1",
+       "text": "Strongly Disagree"
+      },
+      {
+       "value": "2",
+       "text": "Disagree"
+      },
+      {
+       "value": "3",
+       "text": "Slightly Disagree"
+      },
+      {
+       "value": "4",
+       "text": "Neutral"
+      },
+      {
+       "value": "5",
+       "text": "Slightly Agree"
+      },
+      {
+       "value": "6",
+       "text": "Agree"
+      },
+      {
+       "value": "7",
+       "text": "Strongly Agree"
+      }
+     ],
+     "rows": [
+      {
+       "value": "I am going to love teachi",
+       "text": "I am going to love teaching with this curriculum."
+      },
+      {
+       "value": "I understand the objectiv",
+       "text": "I understand the objectives of and the intention behind the curriculum."
+      },
+      {
+       "value": "I understand the teaching",
+       "text": "I understand the teaching strategies I am supposed to use in the curriculum."
+      },
+      {
+       "value": "I understand all of the t",
+       "text": "I understand all of the tools and technologies I am supposed to use in the curriculum."
+      }
+     ]
+    },
+    {
+     "type": "html",
+     "name": "ifCs",
+     "html": "<p><span style=\"background-color: #ffff00;\">&gt;&gt;LOGISTICS AND TOOLS / VIRTUAL-SPECIFIC&lt;&lt;</span></p>\nCurrently a simple FR item (see below).  Could add virtual-specific items here"
```

Comment:
I'm guessing this is a placeholder for later?

Reply:
yes